### PR TITLE
feat: active support for Python 3.13

### PR DESCRIPTION
**Pull request by RobPasMue**
## Description
Provide active support for Python 3.13

## Issue linked
None

## Checklist
- [ ] I have tested my changes locally.
- [ ] I have added necessary documentation or updated existing documentation.
- [ ] I have followed the coding style guidelines of this project.
- [ ] I have added appropriate unit tests.
- [ ] I have reviewed my changes before submitting this pull request.
- [ ] I have linked the issue or issues that are solved to the PR if any.
- [ ] I have assigned this PR to myself.
- [ ] I have added the minimum version decorator to any new backend method implemented.
- [ ] I have made sure that the title of my PR follows [Conventional commits style](https://www.conventionalcommits.org/en/v1.0.0/#summary) (e.g. ``feat: extrude circle to cylinder``)

### CODE REVIEW

**Comment by RobPasMue** [comment]
Missing VTK release compatible with 3.13

File: pyproject.toml [hunk 1]
```diff
@@ -66,7 +67,7 @@ tests = [
     "scipy==1.14.1",
     "semver==3.0.2",
     "six==1.16.0",
-    "vtk==9.3.1",
+    "vtk==9.4.0rc3",
```
Comment (RobPasMue) [inline pyproject.toml:70]:
```suggestion
    "vtk==9.4.0",
```

File: pyproject.toml [hunk 2]
```diff
@@ -104,7 +105,7 @@ doc = [
     "sphinx-copybutton==0.5.2",
     "sphinx-jinja==2.0.2",
     "trame-vtk==2.8.12",
-    "vtk==9.3.1",
+    "vtk==9.4.0rc3",
```
Comment (RobPasMue) [inline pyproject.toml:108]:
```suggestion
    "vtk==9.4.0",
```

**Review by AlejandroFernandezLuces — APPROVED** [review]
LGTM, some considerations:

Should we update the main Python version in the CI file? https://github.com/ansys/pyansys-geometry/blob/9cec7ba847aed85c360f61aaa579a6dbb4af9d4f/.github/workflows/ci_cd.yml#L12

Maybe we could update this link too:

https://github.com/ansys/pyansys-geometry/blob/9cec7ba847aed85c360f61aaa579a6dbb4af9d4f/doc/source/conf.py#L199

**Comment by RobPasMue** [comment]
Looks like everything is working well - I have a lot of numpydoc warnings now with 3.13 but I will fix them soon in a follow-up PR. @AlejandroFernandezLuces would you mind approving?

**Review by AlejandroFernandezLuces — APPROVED** [review]

**Pull request merged**